dvc/remote/gdrive.py

```diff
@@ -83,7 +93,8 @@ def init_drive(self):
             else self.config.get(
                 Config.SECTION_GDRIVE_USER_CREDENTIALS_FILE,
                 os.path.join(
-                    self.repo.tmp_dir, self.DEFAULT_USER_CREDENTIALS_FILE
+                    getattr(self.repo, "tmp_dir", ""),
```

Comment:
Why `getattr` though? `tmp_dir` is always there, it is part of the repo.

Reply:
right, have removed this and added instead into test_gdrive.py following:
```
class Repo(object):
    tmp_dir = ""
```